doc/howto/report-bugs.rst

```diff
@@ -5,6 +5,17 @@ Troubleshooting
 
 This section explains how to deal with installation problems and how to report issues to the Subiquity team.
 
+Validate your autoinstall 'user-data' configuration against the Autoinstall JSON schema :ref:`JSON schema<autoinstall_JSON_schema>`
+----------------
+
+.. code-block:: bash
+
+    git clone https://github.com/canonical/subiquity
+    cd subiquity && make install_deps
+    ./scripts/validate-autoinstall-user-data.py ../ubuntu-autoinstall/user-data
+
+.. note:: The first line in the user-data file have to be "#cloud-config"
```

Comment:
This isn't necessarily true, but is something we're working towards making more clear in the documentation about when it is.

Reply:
was my experience when testing and when i look and the source code.